atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -451,6 +449,14 @@ private static LockAndTimestampServices createRawLeaderServices(
             Supplier<RemoteLockService> lock,
             Supplier<TimestampService> time,
             String userAgent) {
+        if (leaderConfig.leaders().size() == 1) {
```

Comment:
Need to check that the 1 leader is ourself as one may have a single node external timelock service. Should probably add positive & negative test cases for this as well.

```
        if (leaderConfig.leaders().size() == 1
                && leaderConfig.localServer().equals(Iterables.getOnlyElement(leaderConfig.leaders()))) {
```